File: evals/behavioral/tasks/rust/fixtures/safe-index.bad.rs
// Not total: indexes directly and panics out of bounds. The caller receives a
// bare i32 and is never forced to consider the missing element, so
// `get(&v, 99) + 1` type-checks — the invariant is not in the types.
pub fn get(items: &[i32], index: usize) -> i32 {
    items[index]
}

File: evals/behavioral/tasks/scala3/fixtures/safe-index.bad.scala
// Not total: indexes directly and throws IndexOutOfBoundsException out of
// bounds. The caller receives a bare Int and is never forced to consider the
// missing element, so `get(items, 99) + 1` compiles — the invariant is not in
// the types.
def get(items: List[Int], index: Int): Int =
  items(index)
